ibu-imager/internal/seedcreator/seedcreator.go

```diff
@@ -156,7 +157,13 @@ func (s *SeedCreator) stopServices() error {
 	s.log.Println("Stopping containers and CRI-O runtime.")
 	crioSystemdStatus, err := s.ops.SystemctlAction("is-active", "crio")
 	if err != nil {
-		return err
+		exitError, ok := err.(*exec.ExitError)
```

Comment:
Nice @javipolo! Perhaps we could simplify the nested `if`s with something like this:

```go
var exitErr *exec.ExitError
if err != nil && errors.As(err, &exitErr) && exitErr.ExitCode() != 3 {
    return err
}
```

WDYT?

Reply:
I love it @mresvanis ! 
The nested ifs were something I wanted to get rid of, but I couldnt make them go in a nice way ....
Change added and tested! Thanks!